src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -382,7 +401,7 @@ private static final class EnumMap {
      * }
      */
     @SuppressWarnings("removal")
-    private static MethodHandle generateInnerClass(MethodHandles.Lookup caller, Object[] labels) {
+    private static MethodHandle generateInnerClass(MethodHandles.Lookup caller, Class<?> selectorType, Object[] labels) {
```

Comment:
I think the name "labels" for the array here is unfortunate, because the code generation also contains "labels" which mean really "jump targets". Maybe `constants` or `labelConstants` ?

Reply:
Done